plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/sync/SyncNuageVspCmsIdCommand.java

```diff
@@ -17,38 +17,33 @@
 // under the License.
 //
 
-package com.cloud.agent.api.guru;
+package com.cloud.agent.api.sync;
 
 import com.cloud.agent.api.Command;
 
-public class ReleaseVmVspCommand extends Command {
+public class SyncNuageVspCmsIdCommand extends Command {
 
-    String _networkUuid;
-    String _vmUuid;
-    String _vmInstanceName;
+    public enum SyncType { AUDIT, AUDIT_ONLY, REGISTER, UNREGISTER }
 
-    public ReleaseVmVspCommand(String networkUuid, String vmUuid, String vmInstanceName) {
-        super();
-        this._networkUuid = networkUuid;
-        this._vmUuid = vmUuid;
-        this._vmInstanceName = vmInstanceName;
-    }
+    SyncType _syncType;
+    String _nuageVspCmsId;
```

Comment:
Why aren't these attributes declared private? It doesn't appear to modified outside of the constructor. Why aren't they declared final as well?